.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,4 +21,7 @@
     <!-- Generated code should not be subjected to checkstyle. -->
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
+
+    <!-- Exception for SunPkcs1Reader -->
+    <suppress files="SunPkcs1Reader.java" checks="IllegalImport" />
```

Comment:
Too bad. Oh well, let's leave it as it is then?

On Mon, Apr 11, 2016 at 3:29 PM Nick Miyake <notifications@github.com>
wrote:

> In .baseline/checkstyle/checkstyle-suppressions.xml
> <https://github.com/palantir/http-remoting/pull/92#discussion_r59292897>:
>
> > @@ -21,4 +21,7 @@
> >      <!-- Generated code should not be subjected to checkstyle. -->
> >      <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
> >      <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
> > +
> > +    <!-- Exception for SunPkcs1Reader -->
> > +    <suppress files="SunPkcs1Reader.java" checks="IllegalImport" />
>
> Agree that editing Baseline directly is not the best approach. However,
> confirmed that annotation cannot be used to suppress this.
>
> Seems like only work-around would be to enable another suppression filter
> in Baseline, specifically SuppressionCommentFilter or
> SuppressWithNearbyCommentFilter (
> http://checkstyle.sourceforge.net/config_filters.html). With one of these
> enabled, comments can be used to suppress the warnings on the import. This
> would require enabling one of these filters in Baseline's checkstyle.xml.
> Happy to make that change, but wanted to get your feedback before opening a
> PR to do so.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/http-remoting/pull/92/files/778de38d91deecec8d1d8164d58699adb3ffdd41#r59292897>
>